services/datamanager/builtin/capture_test.go

```diff
@@ -383,6 +387,14 @@ func waitForCaptureFilesToEqualNFiles(captureDir string, n int) {
 		}
 
 		time.Sleep(10 * time.Millisecond)
+		if time.Since(start) > 10*time.Second {
+			diagnostics.Do(func() {
+				logger.Infow("waitForCaptureFilesToEqualNFiles diagnostics after 10 seconds of waiting", "numFiles", len(files), "expectedFiles", n)
+				for idx, file := range files {
```

Comment:
I think it'd be helpful to include the dir containing each file. Ideally only one level up since each datacapture file is not super indicative of which collector wrote it as that information is in directory structure, not the file path. If it's not trivial to do that, then I think it's fine without.

Reply:
> Ideally only one level up since each datacapture file is not super indicative of which collector wrote it as that information is in directory structure, not the file path.

Ah, good point! It was easiest to just slap the whole `captureDir` value into the log line. I'm assuming that's also fine:
```
    capture_test.go:378: 2024-06-06T09:55:34.921-0400	INFO		builtin/capture_test.go:378	File information	{"idx":1,"dir":"/tmp/TestFilePolling4193263203/001","name":"2024-06-06T09_55_34.915628176-04_00.capture","size":97}
```